docs/user-guide/config-examples.md

```diff
@@ -39,6 +40,7 @@ all of the nixvim options are accessible no matter how you are using it (flake,
 [GaetanLepage/nix-config]: https://github.com/GaetanLepage/nix-config/tree/master/home/modules/tui/neovim
 [gwg313/nvim-nix]: https://github.com/gwg313/nvim-nix
 [hbjydev/hvim]: https://github.com/hbjydev/hvim
+[khaneliman/khanelivim]: https://github.com/khaneliman/khanelivim
```

Comment:
Unimportant nit: could you keep this list sorted alphanumericly?

I.e. J before K

Reply:
You know I can't read...